fuzz/fuzzers/fuzzer_script_xpm.rs

```diff
@@ -0,0 +1,23 @@
+#![no_main]
+#[macro_use] extern crate libfuzzer_sys;
+extern crate image_extras;
+extern crate image;
```

Comment:
These lines shouldn't be needed anymore

Reply:
Removed; also removed the unnecessary `extern crate` in `fuzzer_script_xbm.rs` and `src/xpm.rs`.